libcxx/test/std/strings/basic.string/string.capacity/resize_and_overwrite.pass.cpp

```diff
@@ -17,6 +17,7 @@
 #include <cassert>
 #include <memory>
 #include <string>
+#include <type_algorithms.h>
```

Comment:
I think it's better to write this as `"type_algorithms.h"` and put this inclusion into the following group.